CSS/style.css

```diff
@@ -413,12 +421,87 @@ img:hover {
   cursor: pointer;
 }
 
-.close {
-  font-size: 45px;
-  font-weight: 600;
+.nav-style > .close {
+  font-size: 60px;
   color: white;
-  margin-top: -300px;
-  float: right;
-  margin-right: 25px;
   cursor: pointer;
+  margin: 20px;
+}
+
+.pop {
+  display: none;
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  max-height: 100vh;
+  width: 100vw;
+  overflow: auto;
+  background: rgb(193 199 208 / 50%);
+  backdrop-filter: blur(3px);
+  padding: 50px 16px;
+  box-sizing: border-box;
+}
+
+.pop-content {
+  background-color: white;
+  box-sizing: border-box;
+  padding: 20px 16px;
+  position: relative;
+  border-radius: 15px;
+}
+
+.head-title {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.close-icon {
+  height: 24px;
+  width: 24px;
+  position: absolute;
+  right: 15px;
+  top: 30px;
+  cursor: pointer;
+}
+
+.work-place {
+  margin-top: 22px;
+  margin-bottom: 20px;
+}
+
+.pop-content > ul > li {
+  list-style: none;
+}
+
+.pop-content > ul > li > .year {
+  color: #344563;
+  font-size: 13px;
+  font-weight: 600;
+}
+
+.popimg {
+  display: flex;
+  position: relative;
+  width: 100%;
+  height: 380px;
+  margin-bottom: 40px;
+}
+
+.popup-buttons {
+  display: grid;
+  grid-template-rows: 1fr;
+  grid-template-columns: 0fr 1fr;
+  column-gap: 12px;
+  border-top: 1px solid #ebecf0;
+  padding-top: 25px;
+}
+
+.popup-buttons > .pop-btn {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 15px;
+  font-weight: 500;
+  line-height: 20px;
```

Comment:
- [ ] Your image in the popup looks a little bit stretched and your popup is higher than the screen making it difficult to see all content inside. Also your tags and buttons are a little bit off from the Figma design. Kindly fix the height, paddings and margins of your elements.
Your mobile design:
<img width="493" alt="image" src="https://user-images.githubusercontent.com/61919617/181076736-91248494-0fb3-44c5-b738-3487c3ecc7c5.png">